bigquery/deploy/bq-indexer-cronjob.yaml.templ

```diff
@@ -0,0 +1,37 @@
+# Use this config to run the indexer as a CronJob on a regular schedule
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: bq-indexer
+spec:
+  # Run daily
+  schedule: "*/3 * * * *"
```

Comment:
Change to once a day

Add comment like `Run every day at 5am UTC`

Reply:
Oops, good catch